src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onChangePage: (page: number, maxCountOfPages?: number) => void;
+  items: string[];
+};
+
+export const Pagination: React.FC<Props> = (
+  {
+    total,
+    perPage,
+    currentPage,
+    onChangePage,
+    items,
+  },
+) => {
+  const firstPage = 1;
+  const lastPage = Math.ceil(total / perPage);
+  const pageItemsList = getNumbers(firstPage, lastPage);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={classNames(
+            'page-item',
+            { disabled: currentPage === firstPage },
+          )}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled="true"
```

Comment:
Write here the boolean value (the same as on the 32nd row) - it will fix tests
And the same for `#prev` link